src/plugin.py

```diff
@@ -226,6 +262,26 @@ def _force_utc(self):
             logging.exception("Can not parse backend response")
             raise UnknownBackendResponse()
 
+        try:
+            steamFolder = get_configuration_folder()
+            accountId = str( int(self._steam_id) - 76561197960265728  )
```

Comment:
I assume this is steamid to steam community id conversion, i think this should be in a separate method then and the '76561197960265728 ' itself should be a const value under a variable